src/axom/primal/operators/detail/intersect_impl.cpp

```diff
@@ -330,35 +330,42 @@ inline bool checkVertex(const Point2& p1,
                         bool includeBoundary,
                         double EPS)
 {
-  if (isGpeq(twoDcross(r2, p2, q1), 0.0, includeBoundary, EPS))
+  // The tests `isGpeq(twoDcross(...))` are checking the orientation
+  // of the triangle defined by its three arguments (CCW vs CW)
+  // Note: Comments in this function refer to regions
+  // in Figure 8 of the paper: https://hal.inria.fr/inria-00072100/document
+
+  /* *INDENT-OFF* */
+
+  if (isGpeq(twoDcross(r2, p2, q1), 0.0, includeBoundary, EPS))                // q1 is in {R_22, R_23, R_24, R_25}
```

Comment:
The comments here are very helpful, but I think it would be easier to read if those comments were in the 1st line inside the if-block. It would also better conform with our style, keep lines at 80 characters, this block of code seems that is not conforming to that.

Reply:
I appreciate the labels out to the side, so they're not squashed in among the code.  The style must allow exceptions to the 80-character rule to serve code legibility, rather than rigidly hindering it.

Reply:
@agcapps -- exceptions are allowed with the `INDENT-OFF`. 

However, we should be careful not to abuse this "feature". Littering the code with exceptions defeats the purpose of having a style in the first place. 

I prefer reading the case labels in the beginning of the code block. But, that is subjective.